spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe ApplicationHelper do
+  describe '#page_title' do
+    context 'given a page that does not exist' do
```

Comment:
Actually is not the page that does not exist, is the `page name`

Reply:
Yes, you're right! I've updated the context description to be clearer, I changed it to:

- `given a page is not defined in the translation file` and
- `given a page is defined in the translation file`

Hopefully this is clearer?